drivers/power/max42500/max42500.h

```diff
@@ -0,0 +1,280 @@
+ *   @file   max42500.h
+ *   @brief  Header file of MAX42500 Driver.
+ *   @author Mark Sapungan (Mark.Sapungan@analog.com)
+********************************************************************************
+ * Copyright 2024(c) Analog Devices, Inc.
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *  - Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *  - Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ *  - Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *  - The use of this software may or may not infringe the patent rights
+ *    of one or more patent holders.  This license does not release you
+ *    from the requirement that you obtain separate licenses from these
+ *    patent holders to use this software.
+ *  - Use of the software either in source or binary form, must be run
+ *    on or directly connected to an Analog Devices Inc. component.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+#ifndef __MAX42500_H__
+#define __MAX42500_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "no_os_i2c.h"
+#include "no_os_gpio.h"
+#include "no_os_timer.h"
+#include "no_os_irq.h"
+
+/** X is set based on the pull configuration of the ADDR pin */
+#define MAX42500_ADDR(x)            (0x28 + (x))
+#define MAX42500_SILICON_ID         (0x30)
+#define MAX42500_I2C_WR_FRAME_SIZE  (4)
+#define MAX42500_I2C_RD_FRAME_SIZE  (5)
+
+/* CONFIG1 bit masks */
+#define MAX42500_CONFIG1_PECE_MASK  NO_OS_BIT(0)
+#define MAX42500_CONFIG1_MBST_MASK  NO_OS_BIT(1)
+#define MAX42500_CONFIG1_RR_MASK    NO_OS_BIT(2)
+
+/* VMON bit masks */
+#define MAX42500_VMON_IN_MASK(bit)  NO_OS_BIT(bit)
```

Comment:
This is just a rename and is not necessary, you can use NO_OS_BIT directly where you configure this VMON_IN